x-pack/plugins/security_solution/server/endpoint/routes/policy/service.ts

```diff
@@ -57,3 +60,70 @@ export async function getPolicyResponseByAgentId(
     policy_response: response.hits.hits[0]._source,
   };
 }
+
+export async function getAgentPolicySummary(
+  endpointAppContext: EndpointAppContext,
+  soClient: SavedObjectsClientContract,
+  packageName: string,
+  policyId?: string,
+  pageSize: number = 1000
+): Promise<JsonObject> {
+  const agentVersionMapToJson = (versionMap: Map<string, number>): JsonObject => {
+    const jsonObject: { [key: string]: number } = {};
+    versionMap.forEach((value, key) => {
+      jsonObject[key] = value;
+    });
+    return jsonObject;
+  };
+
+  const agentQuery = `fleet-agents.packages:"${packageName}"`;
```

Comment:
You should use the `const` from Fleet for the SO type. Import `AGENT_SAVED_OBJECT_TYPE` from `x-pack/plugins/fleet/common` and:

```sugestion
  const agentQuery = `${AGENT_SAVED_OBJECT_TYPE}.packages:"${packageName}"`;
```